src/bootstrap/src/lib.rs

```diff
@@ -1930,8 +1939,8 @@ to download LLVM rather than building it.
     }
 
     /// Checks if the given target is the same as the builder target.
-    fn is_builder_target(&self, target: &TargetSelection) -> bool {
-        &self.config.build == target
+    fn is_builder_target(&self, target: TargetSelection) -> bool {
+        self.config.build == target
     }
```

Comment:
We try very hard in bootstrap to keep `TargetSelection` `Copy`able, noticed while adding some tracing steps.

Reply:
I would rather say that we have to live with the fact that `TargeSelection` is `Copy`able :sweat_smile: